qml/SettingsTranslations.qml

```diff
@@ -0,0 +1,20 @@
+/****************************************************************************
+**
+** Copyright (C) 2013 Jolla Ltd.
+** Contact: Dmitry Rozhkov <dmitry.rozhkov@jolla.com>
```

Comment:
I guess there should not be any contacts here because code is the community patch and Dmitry did not know about this patch ;)